internal/services/springcloud/spring_cloud_service_data_source.go

```diff
@@ -174,7 +174,9 @@ func dataSourceSpringCloudServiceRead(d *pluginsdk.ResourceData, meta interface{
 
 	configServer, err := configServersClient.Get(ctx, id.ResourceGroup, id.SpringName)
```

Comment:
would it not be better to simply not try and get the config server if the sku is `E0`?

Reply:
Sounds better! I'll fix it.